ibis/pyspark/compiler.py

```diff
@@ -1052,32 +1074,39 @@ def compile_window_op(t, expr, scope, timecontext, **kwargs):
     ]
 
     order_by = window._order_by
+    # Timestamp needs to be cast to long for filtering in spark
```

Comment:
"filtering" is not accurate.

Perhaps:
"# Timestamp needs to be cast to long for window bounds in spark"

Reply:
+1